### PR TITLE
Removed duplicated text

### PR DESCRIPTION
**Pull request by IvanBoyko**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by IvanBoyko** [comment]
CLA signed

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -186,19 +184,12 @@ In kubeadm terms, any certificate that would normally be signed by an on-disk CA
 
 ### Create certificate signing requests (CSR)
 
-You can pass in a directory with `--csr-dir` to output the CSRs to the specified location.
-If `--csr-dir` is not specified, the default certificate directory (`/etc/kubernetes/pki`) is used.
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: i don't see this sentence duplicated:
> If `--csr-dir` is not specified, the default certificate directory (`/etc/kubernetes/pki`) is used.

better to move this sentence after:
> Both the CSR and the accompanying private key are given in the output; you can pass in a directory with `--csr-dir` to output the CSRs to the specified location.

Comment (IvanBoyko) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: Agree, will do

Comment (IvanBoyko) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: Done

**Pull request merged**